perses/tests/test_relative.py

```diff
@@ -1110,6 +1110,59 @@ def run_unsampled_endstate_energies(test_name, use_point_energies=True, use_md_e
         vanilla_htf = solvent_delivery.get_apo_htf()
         htfs = [htf, vanilla_htf]
 
+    elif test_name == 'tyk2':
+        import tempfile
+        from perses.utils.url_utils import retrieve_file_url
+        from perses.annihilation.relative import HybridTopologyFactory, RESTCapableHybridTopologyFactory
+        from perses.app.relative_setup import RelativeFEPSetup
+
+        def concatenate_files(input_files, output_file):
+            """
+            Concatenate files given in input_files iterator into output_file.
+            """
+            with open(output_file, 'w') as outfile:
+                for filename in input_files:
+                    with open(filename) as infile:
+                        for line in infile:
+                            outfile.write(line)
```

Comment:
I see this is the same `concatenate_files` from the `run_benchmarks.py` code. Seems like we want this to be in `perses.utils.concatenate_files` and avoid redundant code between both. We would just import it both here and in the benchmarks script.

Reply:
Actually, reading now the whole test, I think what we do really want it to transform the benchmarks code into an actual module in `perses.benchmarks` or similar. This can be left for a future release (next 0.11.0 release sounds like a good option for it). I'll raise an issue with this for the 0.11.0 milestone.

Reply:
Yes, I agree, raising an issue sounds good -- thanks!